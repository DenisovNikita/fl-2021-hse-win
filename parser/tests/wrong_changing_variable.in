begin function main 00
begin body
    var int2 x = 10
    change x :=
        11
end body
end function