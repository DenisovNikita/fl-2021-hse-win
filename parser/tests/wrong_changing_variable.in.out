Error in program.
Expect syntax: 'change <name> = ...', but found: 'change x := ...'
